csv_export.php

```diff
@@ -68,16 +68,8 @@
 
 $t_filename = csv_get_default_filename();
 
-# Send headers to browser to activate mime loading
-
-# Make sure that IE can download the attachments under https.
-header( 'Pragma: public' );
-
-header( 'Content-Type: text/csv; name=' . urlencode( file_clean_name( $t_filename ) ) );
-header( 'Content-Transfer-Encoding: BASE64;' );
-
-# Added Quotes (") around file name.
-header( 'Content-Disposition: attachment; filename="' . urlencode( file_clean_name( $t_filename ) ) . '"' );
+csv_emit_headers( $t_filename );
```

Comment:
I think that nowadays UTF8 encoding is pretty much ubiquitous, and the risk of the receiving end not being able to deal with it is quite low. 

+1 for not adding extra complexity until someone screams for it.